### DISCUSSION TITLE
Kodi error ORANGE

### DISCUSSION THREAD

**Discussion by neptun024** [Paco8/plugin.video.orange.spain#20] [Q&A]
Hola,

Una vez que he realizado toda la configuración que detallas perfectamente, al intentar abrir cualquier canal me salta un error respecto al addon inputstream adaptive, diciéndome que es incompatible con mi versión de Kodi, ¿qué versión utilizas tú?
Muchísimas gracias.

**Comment by Paco8** [comment]
Trata de actualizar el inputstream adaptive. La versión más reciente para kodi 21 es la 21.5.3.

**Comment by Cholo68** [comment]
A mi lo que me ocurre es que al meter la cuenta directamente me salta error de registro y ya aunque quiera entrar al addon no me sale la opcion de añadir cuenta, tengo que desinstalar addon y volver a instalar y a lo mejor me vuelve a ocurrir como que no me ocurre, es aleatorio, estoy con kodi 21.1, una vez que está metida la cuenta el addon funciona perfectamente. Enhorabuena Paco y muchas gracias

P.D. Como no se donde ponerlo aprovecho por aqui, estaria guapisimo que hicieras un addon rtve de estos tuyos, ya se que está l app oficial pero para los que utilizamos todo a traves de kodi estaria genial, no se si habrá alguna posibilidad.

**Comment by Paco8** [comment]
Cuando dé error de registro manda el fichero de registro (el kodi.log) para poder saber qué error da.

**Reply by Cholo68** [reply]
De acuerdo Paco, muchas gracias
